build/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
```

Comment:
I believe this will require changes to the release repo if we are intended to consume these files from this directory

Reply:
i think i need to delete these files.. they were gen'd for me